sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -201,6 +201,14 @@ def get_media_settings_value(physical_port, key):
                         return {}
                 else:
                     return media_dict[key[VENDOR_KEY]]
+            elif key[VENDOR_KEY].split('-')[0] in media_dict:
```

Comment:
The same code addition should be also added to JSONs with 'PORT_MEDIA_SETTINGS' section (right after line 245 in the updated code), and maybe a matching test case in test_xcvrd.py if you find it necessary.

Reply:
I have addressed this now and have added the relevant testcase as well.